skbio/io/format/fasta.py

```diff
@@ -126,23 +127,21 @@
 lines. The sequence data (i.e., nucleotides or amino acids) are stored using
 the standard IUPAC lexicon (single-letter codes).
 
-.. note:: scikit-bio supports both upper and lower case characters. Both ``-``
+.. note:: scikit-bio supports both upper and lower case characters,
+   For reading, this functionality depends on the type of object the data is
+   being read in to. For ``Sequence``
```

Comment:
I did it that way in several places. Are you sure into is correct? Into indicates movement, we're talking about actually placing it inside the object, which deals with location